app/services/users/default_categories_creator.rb

```diff
@@ -0,0 +1,17 @@
+class Users::DefaultCategoriesCreator < Struct.new(:user)
+  def call
+    default_categories_list.map { |name| create_category(name) }
+  end
+
+  private
+
+  def create_category(name)
+    user.categories.create(name: name, amount: 0, user_id: user.id)
+  end
+
+  delegate :categories, to: :user
+
+  def default_categories_list
+    %w[Transportation Food House Entertainment Health Other]
+  end
+end
```

Comment:
@sinpai tests for the service? Don't forget about it

Reply:
Will add them, thanks